warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -1655,16 +1730,68 @@ public void setCollectTimingDetails(boolean collectTimingDetails) {
         this.collectTimingDetails = collectTimingDetails;
     }
 
+    public boolean getCollectTimingDetails() {
+        return collectTimingDetails;
+    }
+
     public void setQuerySpanCollector(QuerySpanCollector querySpanCollector) {
         this.querySpanCollector = querySpanCollector;
     }
 
+    public String toStringNoQueryId() {
+        return toStringImpl(false);
+    }
+
     @Override
     public String toString() {
+        return toStringImpl(true);
+    }
+
+    private String toStringImpl(boolean includeQueryId) {
```

Comment:
make protected

Reply:
comment as to why we have this includeQueryId switch

Reply:
includeQueryId parameter to toStringImpl is so we can have the same logic as in toString() when using this infomation in IvaratorRunnable logs, Ivarator logs, as well as in task names and thread names.  The queryId is part of a standard set of information that will occur in the same order in these places and we don't need to have it there twice.

Reply:
Fixed